### PR TITLE
Tests for axis=0 and 1D input

### PR DESCRIPTION
**Pull request by Deliadd**
mean
slice
unsqueeze(expand_dims)
sum

### CODE REVIEW

**Comment by Deliadd** [comment]
Jenkins passed. ID is 2068
Check the link here: http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Validation/2068/

**Comment by Deliadd** [comment]
Jenkins passed. ID is 2070
Check the link here: http://10.239.47.211:18888/job/ZOO-PR-Validation/2070/

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 1]
```diff
@@ -1430,3 +1431,58 @@ def test_lrn(self):
 
         output = OnnxLoader.run_node(node, [x])
         np.testing.assert_almost_equal(output["y"], y, decimal=5)
+
+    def test_vgg(self):
+        ndarray_input = np.random.randn(1, 3, 224, 224).astype(np.float32)
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: remove this and put it into the document.

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 2]
```diff
@@ -1430,3 +1431,58 @@ def test_lrn(self):
...
+        onnx_model = onnx.load("/home/dingdi/models/vgg.onnx")
+        zmodel = OnnxLoader(onnx_model.graph).to_keras()
+        zmodel.training(is_training=False)
+        zoutput = zmodel.forward(ndarray_input)
+        rep = onnx_caffe2_backend.prepare(onnx_model, device="CPU")  # or "CPU"
+        outputs = rep.run(ndarray_input)
+        self.assert_allclose(zoutput, outputs[0])
+
+    def test_super_resolution(self):
+        x = torch.randn(4, 1, 85, 85, requires_grad=True)
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: remove this and put it into the document.

**Comment by Deliadd** [comment]
Jenkins passed. ID is 2114
Check the link: http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Validation/2114/
